internal/jimm/model.go

```diff
@@ -1347,3 +1351,67 @@ func (j *JIMM) ChangeModelCredential(ctx context.Context, user *openfga.User, mo
 
 	return nil
 }
+
+// ListModels list the models that the user has access to. It intentionally excludes the
+// controller model as this call is used within the context of login and register commands.
+func (j *JIMM) ListModels(ctx context.Context, user *openfga.User) ([]base.UserModel, error) {
```

Comment:
thinking about this implementation a little more, i think it's different from `ListModelSummaries` in an important way.

`ListModelSummaries` uses the models in the db as the source and then uses the controller as another source to merge info before returning the facade. And that's the reason why you don't need to check for uuid to be present in the list of user uuids.
Since our source of truth is always the database and not the openfga tuples, i think you should switch to a similar implementation to `ListModelSummaries`.

Reply:
I'm really not sure what you mean, this call is returning models the user has access to?

Reply:
As discussed, resolving.